modernomad/settings/staging.py

```diff
@@ -0,0 +1,4 @@
+from .common import *  # noqa
+from .common import env
+
+SECRET_KEY = env('SECRET_KEY')
```

Comment:
Well, that depends. Do we want staging settings to be ever so slightly different or at least have the option for that to be the case then yes. If not we could remove staging.py entirely.

Reply:
I don’t think they will be different in any way we can’t change with environment variables. The advantage of reusing the same settings file is staging and production are as close together as possible so we won’t get unexpected bugs in production.

Maybe we want to disable email delivery. I’m not sure whether it is better to do that with envvars or a separate file...

Reply:
Perhaps we can keep it as it is and then we can re-evaluate further down the line?

Reply:
Sure! I’m going to use the production settings for the Heroku staging environment though, because I need the logging and webpack configuration. It seems a bit weird to keep around a staging config that isn’t actually used by staging...